site/topics/logic/combinational-logic.rst

```diff
@@ -10,6 +10,146 @@ Combinational Logic
 Decoders
 ========
 
+* Consider a situation where an input signal controls two outputs, where only one should be active at a time
+
+    * When the input signal is low, one of the two outputs is high
+    * When the input signal is high, the other output should be high
+
+
+.. list-table:: Single Bit Decoder
+    :widths: auto
+    :align: center
+    :header-rows: 1
+
+    * - Input
+      -
+      - Output a
+      - Output b
+    * - ``0``
+      -
+      - ``1``
+      - ``0``
+    * - ``1``
+      -
+      - ``0``
+      - ``1``
+
+
+.. note::
+
+    Until now, all truth tables shown included boolean operators and the tables were completed to describe how the
+    operators act.
```

Comment:
This truth table is still exactly that. This unary operator has two outputs, no biggie. Maybe more precisely is say that all prior truth tables had a single output (...which also isn't really true, since we included columns with sub-problems).

Reply:
rewrote to be more specific about what is meant.